<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="6" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-02-05T18:46:29 GMT+01:00" name="JPASessionUtilTest" finished-at="2022-02-05T18:46:33 GMT+01:00" duration-ms="4066">
    <groups>
    </groups>
    <test started-at="2022-02-05T18:46:29 GMT+01:00" name="Command line test" finished-at="2022-02-05T18:46:33 GMT+01:00" duration-ms="4066">
      <class name="JPASessionUtilTest">
        <test-method signature="getEntityManager()[pri:0, instance:JPASessionUtilTest@10d59286]" started-at="2022-02-05T18:46:29 GMT+01:00" name="getEntityManager" finished-at="2022-02-05T18:46:32 GMT+01:00" duration-ms="3005" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEntityManager -->
        <test-method signature="getSession()[pri:0, instance:JPASessionUtilTest@10d59286]" started-at="2022-02-05T18:46:32 GMT+01:00" name="getSession" finished-at="2022-02-05T18:46:32 GMT+01:00" duration-ms="154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getSession -->
        <test-method signature="nonexistentEntityManagerName()[pri:0, instance:JPASessionUtilTest@10d59286]" started-at="2022-02-05T18:46:32 GMT+01:00" name="nonexistentEntityManagerName" finished-at="2022-02-05T18:46:32 GMT+01:00" duration-ms="8" status="PASS">
          <exception class="javax.persistence.PersistenceException">
            <message>
              <![CDATA[No Persistence provider for EntityManager named nonexistent]]>
            </message>
            <full-stacktrace>
              <![CDATA[javax.persistence.PersistenceException: No Persistence provider for EntityManager named nonexistent
at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:85)
at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
at util.JPASessionUtil.getEntityManager(JPASessionUtil.java:17)
at JPASessionUtilTest.nonexistentEntityManagerName(JPASessionUtilTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- javax.persistence.PersistenceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- nonexistentEntityManagerName -->
        <test-method signature="nonexistentSessionName()[pri:0, instance:JPASessionUtilTest@10d59286]" started-at="2022-02-05T18:46:32 GMT+01:00" name="nonexistentSessionName" finished-at="2022-02-05T18:46:32 GMT+01:00" duration-ms="7" status="PASS">
          <exception class="javax.persistence.PersistenceException">
            <message>
              <![CDATA[No Persistence provider for EntityManager named nonexistent]]>
            </message>
            <full-stacktrace>
              <![CDATA[javax.persistence.PersistenceException: No Persistence provider for EntityManager named nonexistent
at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:85)
at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
at util.JPASessionUtil.getEntityManager(JPASessionUtil.java:17)
at util.JPASessionUtil.getSession(JPASessionUtil.java:23)
at JPASessionUtilTest.nonexistentSessionName(JPASessionUtilTest.java:41)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- javax.persistence.PersistenceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- nonexistentSessionName -->
        <test-method signature="testEntityManager()[pri:0, instance:JPASessionUtilTest@10d59286]" started-at="2022-02-05T18:46:32 GMT+01:00" name="testEntityManager" finished-at="2022-02-05T18:46:33 GMT+01:00" duration-ms="550" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEntityManager -->
        <test-method signature="testSession()[pri:0, instance:JPASessionUtilTest@10d59286]" started-at="2022-02-05T18:46:33 GMT+01:00" name="testSession" finished-at="2022-02-05T18:46:33 GMT+01:00" duration-ms="287" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSession -->
      </class> <!-- JPASessionUtilTest -->
    </test> <!-- Command line test -->
  </suite> <!-- JPASessionUtilTest -->
</testng-results>
